### PR TITLE
added ci as default lab when running pytest

### PR DESCRIPTION
**Pull request by sanderr**
Tests are still failing with the same failure as on master. I still need to look into that.

### CODE REVIEW

File: tests/conftest.py [hunk 1]
```diff
@@ -40,6 +40,7 @@ def pytest_addoption(parser):
     parser.addoption(
         "--lab",
         action="store",
+        default="ci",
```
Comment (edvgui) [inline tests/conftest.py:43]: This means that anyone running the tests locally without being aware of the `--lab` option (which is usually the case until you see the error the first time) will actually use the lab of the ci, potentially creating conflicts, while not being aware of it.  How about rather support setting the option via environment as well and set this one in the ci? (I think the ci templates we have can support this?)

Comment (edvgui) [reply tests/conftest.py:43]: I see that you actually went for the suggestion of @bartv, I don't know how close to it the solution should be.  The current implementation might as well be a temporary patch to fix the ci, and we can implement the env variable support later in a new ticket.

**Comment by sanderr** [comment]
I've made the switch to accept an env var. Could you pay special attention to the correctness of the changelog message and the naming of the env var as I lack the context.

**Review by edvgui — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.0.2 - 03/01/2022
+
+ - Added `INMANTA_TERRAFORM_LAB` environment variable to control testing lab.
```
Comment (edvgui) [inline CHANGELOG.md:5]:
```suggestion
 - Added `INMANTA_TERRAFORM_LAB` environment variable to select testing lab.
```

File: README.md [hunk 3]
```diff
@@ -227,6 +227,7 @@ export GITLAB_TOKEN=""  # A personal gitlab token, with read/write/delete access
 To speed up mutliple test executions, you can set a caching directory so that all the provider binaries don't need to be downloaded every time.  This is done using the `--cache-dir <path-to-folder>` argument.
 
 Finally, the test will need some input, coming from a lab file (in [`tests/labs`](tests/labs/)).  You can base yourself on one of the existing one to create your own.  Then add the `--lab <file-name-minus-.yaml>` argument.
+Alternatively, the lab can be configured through the `INMANTA_TERRAFORM_LAB` environment variable.
```
Comment (edvgui) [inline README.md:230]:
```suggestion
Alternatively, the lab can be picked through the `INMANTA_TERRAFORM_LAB` environment variable.
```

**Comment by sanderr** [comment]
Merged in 2f31f30

**Pull request closed without merging**